config/locales/en.yml

```diff
@@ -121,23 +121,25 @@ en:
       not_signing: signing certificate is needed
       invalid_file_type: is an invalid file type
   profile:
-    title: "User Information"
-    welcome: Welcome
-    heading: "Your Profile"
-    id: "User ID: "
-    given_name: "Given Name: "
-    family_name: "Family Name: "
-    roles: "Roles: "
-    email: "E-mail address: "
-    phone: "Phone number: "
-    team: "Team: "
-    cognito_groups: "Cognito Groups: "
-    permissions_heading: "Your Permissions"
-    client_switch_heading: "Switch Authentication Clients"
-    client_switch_legend: "In development you can switch between using cognito and the stub authentication clients. If you use the stub client you can assume different roles."
-    banner: "There are features on this page which are only avialable in the development environment."
-    assume_roles_heading: "Switch Roles"
-    assume_roles_legend: "Select which role(s) you wish to assume?"
+    title: Your profile
+    item: Item
+    details: Your details
+    action: Action
+    name: Name
+    email: Email Address
+    password: Password
+    last_changed: Last changed X days ago
```

Comment:
we don't have this functionality and will not be available in the near future, can we just drop it?